### DISCUSSION TITLE
How to restart footnote numbering on each chapter in article?

### DISCUSSION THREAD

**Discussion by kcho17** [jgm/pandoc#9839] [Q&A] [answered]
I want to restart footnote number on each chapter in ``article`` document class. I learned that this can be done only with ``book`` or ``report`` class. But is there any way that it can be done with the article class?

**Comment by TomBener** [comment, marked as answer]
The issue lies in `\chapter` command is not available for the `article` class. Instead, the command is only available in the `report` or `book` class. You can see the answer on [LaTeX Stack Exchange](https://tex.stackexchange.com/a/604928/176776).

**Reply by kcho17** [reply]
I see. Is there any way I can restart fn numbering without using the command ``\chapter`` command then? I tried the solution that you directed me but it didn't work...

**Reply by TomBener** [reply]
Try these:

- [Reset footnote counter every section](https://tex.stackexchange.com/questions/315932/reset-footnote-counter-by-section-with-footmisc)
- [Reset footnote counter per page](https://tex.stackexchange.com/questions/1656/footnote-counter-would-like-to-restart-from-1-each-page)

**Reply by kcho17** [reply]
Yay it works! Thanks!
